fcrepo-client-impl/src/main/java/org/fcrepo/client/impl/FedoraResourceImpl.java

```diff
@@ -160,9 +253,45 @@ public Long getSize() {
     }
 
     @Override
-    public void move(final String destination) throws ReadOnlyException {
-        // TODO Auto-generated method stub
-        throw new NotImplemented("Method move(final String destination) is not implemented.");
+    public void move(final String destination) throws FedoraException {
+        final HttpMove move = httpHelper.createMoveMethod(path,destination);
+
+        try {
+            final HttpResponse response = httpHelper.execute( move );
+            final StatusLine status = response.getStatusLine();
+            final String uri = move.getURI().toString();
+
+            if (status.getStatusCode() == HttpStatus.SC_CREATED) { // Created
+                LOGGER.debug("resource successfully moved from " + path + " to " + destination, uri);
+                removeTombstone();
```

Comment:
To be consistent with the delete() behavior, I don't think the tombstone should be removed automatically.  So I think we should either have a forceMove() method that moves and removes the tombstone, or maybe add an argument to both delete() and move() to control whether the tombstone is deleted or not.